modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,75 +1,81 @@
+AS2 endpoint settings dictate where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
 
-Create the AS2 identifiers for your organization and your trading partners in the respective host and trading partner profile pages.
+You can create the AS2 identifiers for your organization and for your trading partners in the respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[trading partner] profile pages.
 
 [%header%autowidth.spread]
 |===
-|Setting |Description | Required
-|Endpoint Name
-|Name used to easily identify the purpose of this endpoint. The name is auto-populated, but can be overwritten. 
-|Yes 
+| Setting | Description | Required
+| Endpoint Name
+| The name used to identify the endpoint. The `Endpoint Name` is automatically generated, but can be modified manually.
+| Yes
 
-|Description
-|Describe any important details for this endpoint. 
+| Owner
+| The trading partner or host that owns the endpoint. 
+| Yes
+
+| Protocol
+| The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.
+| Yes
+
+| Description
+| The user-defined field for describing the purpose and use of the endpoint.
 | No
 
 | AS2 URL
 a| The HTTP or HTTPS address where you receive AS2 messages.
 
-When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint.
-You can see the URL  by selecting the receiving endpoint from your deployed message flow.
-
-Select the receiving endpoint from your deployed message flow to view the URL.
+When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint. To view the URL, select the receiving endpoint from your deployed message flow.
 
-If you are deploying to on-premises Mule with a firewall and load balancer, work with your organization's hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is executing. You must replace the {runtime_host} value in the URL with the value that is specific to your organization. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is: `+https://b2b.mycompany.com/receive-as2/+`
-|
+If you are deploying to on-premises Mule server with a firewall and load balancer, work with your organization's hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is executing. You must replace the {runtime_host} value in the URL with the value that is specific to your organization. For example, if the DNS host name is `b2b.mycompany.com`, the AS2 receiver URL is: `+https://b2b.mycompany.com/receive-as2/+`
+| No
 
 | <Host> public certificate
```

Comment:
As noted above, is this syntax/formatted as syntax?

Reply:
In this instance yes - this is user input.